Makefile

```diff
@@ -80,8 +80,7 @@ ifdef LLAMA_SERVER_VERBOSE
 endif
 
 # warnings
-MK_CFLAGS   += -Wall -Wextra -Wpedantic -Wcast-qual -Wdouble-promotion -Wshadow -Wstrict-prototypes -Wpointer-arith \
-               -Wmissing-prototypes -Werror=implicit-int
+MK_CFLAGS   += -Wall -Wextra -Wpedantic -Wcast-qual -Wdouble-promotion -Wshadow -Wstrict-prototypes -Wpointer-arith -Wmissing-prototypes -Werror=implicit-int
```

Comment:
I usually like to keep code under 120 characters wide, for the sake of readability.

Reply:
Yup, I didn't notice the first version had spaces and thought the tabs are causing the problem. Should be ok now